SplitFiles/bread.h

```diff
@@ -33,26 +23,40 @@ struct BreadTypeStruct {
 };
 typedef struct BreadTypeStruct BreadType;
 
+struct BreadStateStruct {
+  int               temperature;  
+  int               moisture;
+  int               toastedness;
+  unsigned char     thermalMass;
+  unsigned int      thermalAggregation;
+  BreadType*        type; // keep reference to underlying BreadType, for letter, desc
+};
+
+typedef struct BreadStateStruct BreadState;
+
 
 struct BreadBinStruct {
     int probTotal;
     BreadType** breadTypes;                    //will be an array of 26 pointers 
                                                //       where (BreadTypeStruct.letter - 'A') is the index
```

Comment:
Method pointers in the struct would be my preference but it's just too fiddly sometimes!